### PR TITLE
Feature chipimp refactor

### PR DESCRIPTION
**Pull request by anastasia-shch**
In this update chip/imputed data switch is refactored in such a way that the switch appears only on the main variant view page. Fetching of the data from server is done now automatically once the genotype source is re-selected. In addition, gene and range search table are modified to contain an additional column with genotype source. Server side was updated to support this change.

### CODE REVIEW

File: react/features/variant/VariantForm.js [hunk 1]
```diff
@@ -105,9 +110,7 @@ export const VariantForm = (props) => {
 		}
 	}
 
-	const hide_imputchip_radio = dtype != undefined ?
-		dtype == 'chip' ? true : false
-	: false
+	const hide_imputchip_radio = dtype != null ? dtype == 'chip' ? true : false : false
```
Comment (juhis) [inline react/features/variant/VariantForm.js]: Can this be just const hide_imputchip_radio = dtype == 'chip' ?

**Review by juhis — COMMENTED** [review]
This is great, simplifies things and I also think is better for the user. There are some conflicts with the current master - would it be possible to merge the current master to this first resolving conflicts so we can then merge this to master?

File: react/features/variant/VariantForm.js (hunk 1 above)
Comment (anastasia-shch) [reply react/features/variant/VariantForm.js]: Makes more sense,  thanks

Comment (anastasia-shch) [reply react/features/variant/VariantForm.js]: Thank you for the comments! Merged as suggested current master to the branch and resolved conflicts.

**Pull request merged**